app-dicts/myspell-en/myspell-en-20181101.ebuild

```diff
@@ -25,6 +25,7 @@ src_prepare() {
 		MYSPELL_HYPH=( "hyph_en_GB.dic" )
 	fi
 	if use l10n_en-US || use l10n_en; then
+		MYSPELL_HYPH=( "hyph_en_US.dic" )
```

Comment:
(Noticed while testing that this causes file collission with <hyphen-2.8.8-r1, maybe add a soft blocker and revbump this ebuild, too?)

Reply:
Ok I added a soft block in myspell requiring hyphen-2.8.8-r1 or newer to be installed

Reply:
And I'm sorry to say this, but it requires a revbump, as I hinted earlier.

Reply:
ok also added revbump, sorry missed that.